conf/modules.config

```diff
@@ -231,76 +261,106 @@ process {
+        ]
     }
     withName: 'SAMTOOLS_MERGE_CRAM' {
         ext.suffix       = '.recal'
         publishDir       = [
-            path: { "${params.outdir}/preprocessing/${meta.id}/recalibrated" },
             enabled: true,
+            mode: 'copy',
+            path: { "${params.outdir}/preprocessing/${meta.id}/recalibrated" },
             pattern: "*cram"
         ]
     }
     withName: 'QUALIMAP_BAMQC_CRAM' {
         ext.args         = '--paint-chromosome-limits --genome-gc-distr HUMAN -skip-duplicated --skip-dup-mode 0 -outformat HTML'
         ext.suffix       = '.recal'
         publishDir       = [
-            path: { "${params.outdir}/reports/qualimap/${meta.id}" },
-            enabled: true
+            enabled: true,
+            mode: 'copy',
+            path: { "${params.outdir}/reports/qualimap/${meta.id}" }
         ]
     }
     withName: 'INDEX_RECALIBRATE' {
         ext.suffix       = 'recal'
         publishDir       = [
-            path: { "${params.outdir}/preprocessing/${meta.id}/recalibrated" },
             enabled: true,
+            mode: 'copy',
+            path: { "${params.outdir}/preprocessing/${meta.id}/recalibrated" },
             pattern: "*{recal.cram,recal.cram.crai}"
         ]
     }
     withName: 'SAMTOOLS_STATS' {
         publishDir       = [
-            path: { "${params.outdir}/reports/samtools_stats/${meta.id}" },
-            enabled: true
+            enabled: true,
+            mode: 'copy',
+            path: { "${params.outdir}/reports/samtools_stats/${meta.id}" }
         ]
     }
 }
 
-// // GERMLINE_VARIANT_CALLING
-// withName: haplotypecaller {
-//     ext.args         = '-ERC GVCF'
-//     ext.suffix       = '.g'
-//     publishDir       = [
-//             // path: { "${params.outdir}/variant_calling'
-//     publish_files    = false
-// }
-// withName: genotypegvcf {
-//     publishDir       = [
-//             // path: { "${params.outdir}/variant_calling'
-//     publish_files    = false
-// }
-// withName: concat_haplotypecaller {
-//     ext.suffix       = 'haplotypecaller'
-//     publishDir       = [
-//             path: { "${params.outdir}/variant_calling'
-//     publish_files    = ['vcf.gz':'haplotypecaller', 'vcf.gz.tbi':'haplotypecaller']
-// }
-// withName: concat_gvcf {
-//     ext.suffix       = 'haplotypecaller_gvcf'
-//     publishDir       = [
-//             path: { "${params.outdir}/variant_calling'
-//     publish_files    = ['vcf.gz':'haplotypecaller_gvcf', 'vcf.gz.tbi':'haplotypecaller_gvcf']
-// }
-// withName: strelka_germline {
-//     publishDir       = [
-//             path: { "${params.outdir}/variant_calling'
-//     publish_files    = ['vcf.gz':'strelka', 'vcf.gz.tbi':'strelka']
-// }
-// withName: deepvariant {
-//     ext.args         = '--model_type=WGS'
-//     publishDir       = [
-//             path: { "${params.outdir}/variant_calling'
-//     publish_files    = false
-// }
+// GERMLINE_VARIANT_CALLING
+process {
```

Comment:
This was commented out, because @GCJMackenzie is adding in a new subworkflow that also allows joint calling. @GCJMackenzie maybe you can take a look at the gatk based tools and their parameters.